Src/java/qdm/src/main/resources/gov/healthit/qdm/qdm-modelinfo-5.3.xml

```diff
@@ -257,6 +269,7 @@
         <ns4:element name="ordinality" type="System.Code"/>
         <ns4:element name="incisionDatetime" type="System.DateTime"/>
         <ns4:element name="negationRationale" type="System.Code"/>
+        <ns4:element name="component" type="list&lt;QDM.Component&gt;"/>
```

Comment:
Yes, this is a change applied by the other QDM-related pull request, I must have inadvertently copied the source from that PR. Correcting.